Documentation~/en/install.md

```diff
@@ -0,0 +1,38 @@
+# Install package
+
+> [!NOTE]
+> Unity 2020.1 and 2019.4 are difference how to install the WebRTC package so please attention the Unity version you are using and follow the instruction below.
```

Comment:
In Unity 2020.1 and 2019.4, there are differences about how to install WebRTC package so please pay attention to the Unity version you are using and follow the instructions below.

Reply:
Thanks!